### ISSUE TITLE
[Request] Can we add the 2022 GSoC accepted proposals

### ISSUE THREAD

**Issue by jaiakash** [Google-Summer-of-Code-Archive/gsoc-proposals-archive#40]
Since GSoC 2022 is ended, can we start adding the proposals? It would be helpful to 2023 contributors.

**Comment by cybergeekgyan** [comment]
I have added the GSOC 2022 Accepted proposals for the respective organisations and will be adding more.. DO contribute if you find any proposals for 2022 do add it here

**Comment by IshitaPathak** [comment]
hey @jaiakash. Can I work on this issue?
I have some GSOC 2022 accepted proposals

**Comment by cybergeekgyan** [comment]
@IshitaPathak I don't think This is active, I have opened a PR for 2022 accepted proposals but no action on  the PR yet

**Comment by IshitaPathak** [comment]
@cybergeekgyan Oh,OK! Thanks for letting me know.

**Comment by shubhank-saxena** [comment]
Hello folks. I just merged all of your PRs. 
Kindly circulate the repo link amongst 2022 folks that you know of!
